tools/PI/DevHome.PI/Helpers/ExternalToolsHelper.cs

```diff
@@ -36,65 +68,161 @@ private ExternalToolsHelper()
             localFolder = Path.GetDirectoryName(Assembly.GetEntryAssembly()?.Location) ?? string.Empty;
         }
 
+        // The file should be in this location:
+        // %LocalAppData%\Packages\Microsoft.Windows.DevHome.Dev_8wekyb3d8bbwe\LocalState\externaltools.json

```

Comment:
Suggest not using Dev package path
```suggestion
        // %LocalAppData%\Packages\Microsoft.Windows.DevHome_8wekyb3d8bbwe\LocalState\externaltools.json
```

Reply:
Fixed in 2nd commit.